test/io/__snapshots__/test_cli/test_schema_cache_snapshot[dbMediaHandlers].yaml

```diff
@@ -3,11 +3,6 @@
   - - tag: BuiltinOvAggCsv
     - tag: MTTextCSV
 
-- - - tag: RelAnyElement
-    - tag: MTGeoJSON
-  - - tag: BuiltinOvAggGeoJson
-    - tag: MTGeoJSON
-
```

Comment:
The `test/io/fixures.sql` don't have PostGIS enabled, hence why these lines are deleted. I think these dbMediaHandlers make sense as a default.

`test/spec/Feature/Query/PostGISSpec.hs` already proves that PostGIS works as always.